script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -255,7 +260,7 @@ I was glad that in Keiko-san's mind, I was still her child.
 [sha:08bb258065cad0ce33ed2594bf11ec4d6e83c8b1]{
 -- Page 19, Offset 1195.
 -- 行ってらっしゃい、と返さないのが啓子さんらしい。
-It was typical of Keiko-san not to reply with something like 'Have a safe trip.'
+That Keiko-san didn't send me off with a 'see you' was just like her.
```

Comment:
I don't love `just like her`, it's one of those phrases that sticks out as being used a lot in jp but not really in en and can benefit from being munged a bit.

Reply:
Just noticed this now but the repeated 'sent off' in the next sentence sounds awkward, I would keep the 'reply' in this one. 

> That Keiko-san didn't reply with a 'see you' was something I'd expected.